src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2177,7 +2177,7 @@ Copied_version_to_clipboard=Copied_version_to_clipboard
 
 BibTeX_key=BibTeX_key
 Message=Message
-
+ID-based_entry_generator=ID-based_entry_generator
```

Comment:
The order of entries in `JabRef_en.properties` is important. Please move this together with `No_entry_with_id_'%0'_for_fetche...`. Maybe at the end of the file (as it is common for new entries)